src/commands/targeting/enable.ts

```diff
@@ -45,10 +47,12 @@ export default class EnableTargeting extends Base {
         if (flags.headless) {
             this.writer.showResults(updatedTargeting)
```

Comment:
not related to this PR but do you know why we're not using the tree view in headless mode? we have all the required info to do it

Reply:
ah okay